### DISCUSSION TITLE
Multiple Y axis support

### DISCUSSION THREAD

**Discussion by meckyp** [chartjs/chartjs-plugin-zoom#489] [Q&A] [answered]
Is it possible to set a zoom range on the secondary Y-axis?

**Comment by kurkle** [comment]
#478 added `zoomScale` function that can be used to manipulate any one scale of the chart.

Its available in v1.0.0-beta.4

**Comment by kurkle** [comment, marked as answer]
#524 added the ability to set limits by scale id. Its available since v1.1.0
